dashboard/test/ui/features/support/connect.rb

```diff
@@ -7,18 +7,19 @@
 require 'retryable'
 
 UI_TEST_DIR = File.expand_path('../..', __dir__)
-$browser_config = JSON.parse(File.read(File.join(UI_TEST_DIR, "browsers.json"))).detect {|b| b['name'] == ENV['BROWSER_CONFIG']} || {}
+$browser_config = JSON.parse(File.read(File.join(UI_TEST_DIR, 'browsers.json'))).detect {|b| b['name'] == ENV['BROWSER_CONFIG']} || {}
 
 MAX_CONNECT_RETRIES = 3
+SAUCELABS_SELENIUM_URL = 'https://ondemand.us-west-1.saucelabs.com/wd/hub'.freeze
```

Comment:
I wouldn't hate if this was over-rideable by an env var. Not a must-have, but would allow say a 3rd party to use their suacelabs endpoint which happened to be on us-east-1 or whatever.